libc/test/src/math/CMakeLists.txt

```
@@ -44,6 +44,63 @@ add_fp_unittest(
     libc.src.__support.FPUtil.fp_bits
 )
 
+
+add_fp_unittest(
+  daddl_test
+  NEED_MPFR
+  SUITE
+    libc-math-unittests
+  SRCS
+    daddl_test.cpp
+  HDRS
+    AddTest.h
+  DEPENDS
+    libc.src.math.daddl
+    libc.src.__support.FPUtil.generic.add_sub
```

The unit tests themselves don't use `libc.src.__support.FPUtil.generic.add_sub`.

```suggestion
  DEPENDS
    libc.src.math.daddl
```

---

This wasn't resolved.